channeldb/migration25/channel.go

```diff
@@ -720,3 +720,39 @@ func fetchChannelLogEntry(log kvdb.RBucket,
 	commitReader := bytes.NewReader(commitBytes)
 	return mig.DeserializeChanCommit(commitReader)
 }
+
```

Comment:
Typo in commit message: severl -> several

Reply:
fixed!